k2/python/k2/fsa_algo.py

```diff
@@ -920,3 +920,60 @@ def prune_on_arc_post(fsas: Fsa, threshold_prob: float,
     out_fsa = k2.utils.fsa_from_unary_function_tensor(fsas, ragged_arc,
                                                       arc_map)
     return out_fsa
+
+
+def expand_ragged_labels(
```

Comment:
Shall we change it to `expand_ragged_attributes`?

The `labels` of an Fsa is always a normal tensor. Only the attributes
may be ragged.